microsoft/botframework-solutions
# [Enterprise Bot Template][TS] Add TSLint Tool as dependency

## Description
- Add **TSlint** as dependency.
- Add rules into `tsconfig.json` file.
- Add `npm run lint` into `build` script.

## Related Issue
Fixes [#563](https://github.com/Microsoft/AI/issues/563) 

## Testing Steps
1. Open a new terminal in `AI/templates/Enterprise-Template/src/typescript`.
2. Run `npm install` to install dependencies.
3. Run `npm run build` to build the solution.
4. Check the execution output of tslint in terminal.

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
